src/main/java/au/com/agiledigital/idea_search/dao/FedexIdeaDao.java

```diff
@@ -26,255 +23,272 @@
-     *
-     * @param technology string
-     * @return AoFedexTechnology object to be saved to the data store
-     */
-    private AoFedexTechnology prepareAOFedexTechnology(String technology) {
-        AoFedexTechnology aoFedexTechnology =
-                this.ao.create(AoFedexTechnology.class);
-        aoFedexTechnology.setTechnology(technology);
-        return aoFedexTechnology;
-    }
-
-    /**
-     * Prepare fedex active object with the data from a fedex idea
-     *
-     * @param aoFedexIdea active object
-     * @param fedexIdea   with data to be added to the active object
-     */
-    private void prepareAOFedexIdea(
-            AoFedexIdea aoFedexIdea,
-            FedexIdea fedexIdea
-    ) {
-        aoFedexIdea.setContentId(fedexIdea.getContentId());
-        aoFedexIdea.setCreatorUserKey(this.getUserKey(fedexIdea.getCreator()));
-        aoFedexIdea.setOwner(fedexIdea.getOwner());
-        aoFedexIdea.setStatus(fedexIdea.getStatus());
-        aoFedexIdea.setDescription(fedexIdea.getDescription());
-    }
-
-    /**
-     * Convert fedex idea active object to a fedex idea model object
-     *
-     * @param aoFedexIdea active object to be converted
-     * @return FedexIdea object
-     */
-    private FedexIdea asFedexIdea(AoFedexIdea aoFedexIdea) {
-        return aoFedexIdea == null
-                ? null
-                : (
-                new au.com.agiledigital.idea_search.model.FedexIdea.Builder()
-        ).withGlobalId(aoFedexIdea.getGlobalId())
-                .withOwner(aoFedexIdea.getOwner())
-                .withContentId(aoFedexIdea.getContentId())
-                .withCreator(this.getUsername(aoFedexIdea.getCreatorUserKey()))
-                .withDescription(aoFedexIdea.getDescription())
-                .withStatus(aoFedexIdea.getStatus())
-                .build();
-    }
-
-    /**
-     * Convert fedex technology active object to a fedex technology model object
-     *
-     * @param aoFedexTechnology active object to be converted
-     * @return FedexTechnology object
-     */
-    private FedexTechnology asFedexTechnology(
-            AoFedexTechnology aoFedexTechnology
-    ) {
-        return aoFedexTechnology == null
-                ? null
-                :
-                new au.com.agiledigital.idea_search.model
-                        .FedexTechnology.Builder()
-                        .withGlobalId(aoFedexTechnology.getGlobalId())
-                        .withTechnology(aoFedexTechnology.getTechnology())
-                        .build();
+  }
+
+  /**
+   * Convert a user key ID to the users name
+   *
+   * @param userKey string of the user key id
+   * @return userName string
+   */
+  private String getUsername(String userKey) {
+    if (userKey == null) {
+      return null;
+    } else {
+      ConfluenceUser user =
+        this.userAccessor.getUserByKey(new UserKey(userKey));
+      return user == null ? null : user.getLowerName();
     }
+  }
```

Comment:
I feel like inverting this if statement would flow better.

```suggestion
  private String getUsername(String userKey) {
    if (userKey != null) {
      ConfluenceUser user =
        this.userAccessor.getUserByKey(new UserKey(userKey));
      return user == null ? null : user.getLowerName();
    }
    
    return null;
  }
```

Reply:
Updated